sample-compose/app/src/main/java/co/nimblehq/sample/compose/extensions/FlowExt.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.sample.compose.extensions
+
+import android.annotation.SuppressLint
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import kotlinx.coroutines.flow.*
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+
+@SuppressLint("ComposableNaming")
+@Composable
+fun <T> Flow<T>.collectAsEffect(
+    context: CoroutineContext = EmptyCoroutineContext,
+    block: (T) -> Unit,
```

Comment:
Can be improved a little bit so that we can also run suspend functions like `delay()`
```suggestion
    block: suspend (T) -> Unit,
```

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/449/commits/618fd7d9f2031c8695ee58ee2f6d59a644f1b7a2